lib/runBlock.js

```diff
@@ -84,7 +85,7 @@ module.exports = function (opts, cb) {
     function processTx (tx, cb) {
       var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).cmp(new BN(tx.gasLimit).add(gasUsed)) === -1
       if (gasLimitIsHigherThanBlock) {
-        cb('tx has a higher gas limit than the block')
+        cb(new Error('tx has a higher gas limit than the block'))
```

Comment:
I think it all of the callback errors should be of the Error class, but prefer to do it in a separate PR - there might be other cases too.